uber/config.py

```diff
@@ -111,6 +111,15 @@ def get_attendee_price(self, dt):
     def get_group_price(self, dt):
         return self.get_attendee_price(dt) - self.GROUP_DISCOUNT
 
+    def get_badge_count_by_type(self, badge_type):
```

Comment:
I would LOVE a @derived_property property. The `__getattr__` methods are really cool and useful, but having them clumped together makes them hard to read.